src/cdp/router.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { StubServer } from '../tests/stubServer.spec';
+
+import * as chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+chai.use(chaiAsPromised);
+
+import * as sinon from 'sinon';
+
+import { MessageRouter } from './router';
+
+describe('MessageRouter', function () {
+  it('can send a message without a sessionId', async function () {
+    const mockCdpServer = new StubServer();
+    const router = new MessageRouter(mockCdpServer);
+
+    const message = JSON.stringify({ id: 0, method: 'Browser.getVersion' });
+    router.sendMessage(message);
+
+    sinon.assert.calledOnce(mockCdpServer.sendMessage);
+    sinon.assert.calledWith(mockCdpServer.sendMessage, message);
+  });
+
+  it('can send a message with a sessionId', async function () {
```

Comment:
Yeah, the Router class doesn't currently care about sessionIds when sending commands. It just forwards everything to the backend as-is. I don't think the Router will need any extra logic here, but I'd like to keep both tests here in case that changes in the future.

I'll merge these into one test though since both cases are essentially the same.